frontend/src/notifications/dtos/notification-event/OrganizationCreatedEventDto.ts

```diff
@@ -0,0 +1,9 @@
+import { NotificationEvent } from '@notifications/enums/notification-event';
+import { NotificationEventDto } from './NotificationEventDto';
+
+export interface OrganizationCreatedEventDto
+  extends NotificationEventDto<NotificationEvent, Record<string, string>> {
```

Comment:
Tu sa 2 generyki, ktore totalnie nic nie robia. Wydaje mi sie, ze to powinno wygladac w ten sposob

```
export interface NotificationEventDto<TData extends Record<string, unknown>> {
  event: NotificationEvent;
  data: TData;
}
```

```
export interface OrganizationCreatedEventDataDto {
  organizationId: string;
}

export interface OrganizationCreatedEventDto extends NotificationEventDto<OrganizationCreatedEventDataDto>;
```